kornia/augmentation/augmentation.py

```diff
@@ -1346,13 +1486,20 @@ def __init__(self, kernel_size: Tuple[int, int],
                  p: float = 0.5) -> None:
         super(GaussianBlur, self).__init__(
             p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.)
-        self.transform = GaussianBlur2d(kernel_size, sigma, border_type)
+        self.kernel_size = kernel_size
+        self.sigma = sigma
+        self.border_type = BorderType.get(border_type)
 
     def __repr__(self) -> str:
         return self.__class__.__name__ + f"({super().__repr__()})"
 
     def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor]:
```

Comment:
We should be able too skip implementing this from users side

Reply:
Probably